README.md

```diff
@@ -12,6 +12,18 @@ Scuttlebutt is a decent(ralised) secure gossip platform.
 >
 > Or, if you're lazy: http://en.wikipedia.org/wiki/Scuttlebutt (laziness will get you nowhere, btw)
 
+## Secure-scuttlebutt vs scuttlebutt vs scuttlebot vs sbot?
+
+* scuttlebutt: a gossip protocol that synchronises messages via a vector clock of per-node timestamp or sequences. I got this name from an amazon paper "Efficient Reconciliation and Flow Control for Anti-Entropy Protocols". Also my original scuttlebutt module, which should now be called "insecure scuttlebutt"
```

Comment:
Formatting: sure
"my original": I copied and pasted from a question by Aljoscha and answer by Dominic on the patchwork network. So it was said by Dominic :)